README.Fedora

```diff
@@ -14,9 +14,35 @@ sudo dnf install cmake gcc-g++ libevent-devel golang autoconf pkgconfig \
     json-c-devel gflags-devel glog-devel protobuf-devel leveldb-devel \
```

Comment:
This doc now only applies if you're not using recommended gclient build so I think it's now OK.